x-element.js

```diff
@@ -118,8 +118,7 @@ export default class XElement extends HTMLElement {
 
   /** Extends HTMLElement.prototype.connectedCallback. */
   connectedCallback() {
-    XElement.#initializeHost(this);
-    XElement.#addListeners(this);
+    XElement.#connectHost(this);
```

Comment:
I wrapped these simply for readability / symmetry. All of these are private, so it's not like we're making public interface changes here.